tools/appveyor.psm1

```diff
@@ -326,9 +326,15 @@ function Invoke-AppVeyorTest
     Write-Host -Foreground Green 'Upload FullCLR test results'
     Update-AppVeyorTestResults -resultsFile $testResultsFileFullCLR
 
-    ## CodeCoverage
-    $env:CodeCoverageOutput = Split-Path -Parent (Get-PSOutput -Options (New-PSOptions -Configuration CodeCoverage))
- 
+    if(Test-DailyBuild)
```

Comment:
Chat with @adityapatwardhan offline and we think this logic can be wrapped into a function and be called in the `after_test:` phase.